Malmo/src/ClientConnection.cpp

```diff
@@ -28,85 +28,117 @@
 
 namespace malmo
 {
-    boost::shared_ptr< ClientConnection > ClientConnection::create( boost::asio::io_service& io_service, std::string address, int port )
+    boost::shared_ptr<ClientConnection> ClientConnection::create(boost::asio::io_service& io_service, std::string address, int port)
     {
-        return boost::shared_ptr< ClientConnection >(new ClientConnection( io_service, address, port ) );
+        return boost::shared_ptr<ClientConnection>(new ClientConnection(io_service, address, port));
     }
-    
-    void ClientConnection::send( std::string message )
+
+    void ClientConnection::send(std::string message)
     {
-        //LOGTRACE(LT("Request to send "), message, LT(" to "), this->)
-        this->io_service.post( boost::bind( &ClientConnection::writeImpl, shared_from_this(), message ) );
+        this->io_service.post(boost::bind(&ClientConnection::writeImpl, shared_from_this(), message));
     }
 
-    ClientConnection::ClientConnection( boost::asio::io_service& io_service, std::string address, int port )
-        : io_service( io_service )
+    ClientConnection::ClientConnection(boost::asio::io_service& io_service, std::string address, int port)
+        : io_service(io_service)
     {
+        // DEBUG FAKE 
+        // std::cout << "CONNECT SIT DOWN" << std::endl;
+        // std::this_thread::sleep_for(std::chrono::seconds(1000));
+
```

Comment:
Do we need to keep this lying around? It's in the git history if we need to refer back to it.

Reply:
Oops, this was test code for agent being unresponsive. Will remove.